packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -15,7 +15,7 @@ import { __, sprintf } from '@wordpress/i18n';
  */
 import { getPasteEventData } from '../../utils/get-paste-event-data';
 
-function useNotifyCopy() {
+export function useNotifyCopy() {
```

Comment:
With `useNotifyCopy` being used in more than one component, is `copy-handler` still the right place for its definition? (Also, is this hook a good abstraction to begin with? 🙂)

Reply:
Hey Miguel, thanks for the feedback!

`useNotifyCopy` could certainly go to a different file, under `copy-handler` folder ( even if it wasn't reused ) and maybe renamed to reflect that handles cut notifications too.

I believe it's a good abstraction as it does one thing: notifies the user of a copy/cut action. Every such action should have consistency about the way a user is notified.